Jenkinsfile

```diff
@@ -426,6 +448,30 @@ pipeline {
                 }
               }
             }
+            stage('Torch GPU Based on Nvidia Image') {
+              agent {
+                docker {
+                  label "linux-gpu-node"
+                  image "nvcr.io/nvidia/pytorch:22.04-py3"
+                  args "--runtime nvidia --shm-size=8gb"
+                  alwaysPull false
+                }
+              }
+              stages {
+                stage('Torch GPU Unit test') {
+                  steps {
+                    sh 'nvidia-smi'
+                    sanity_check_for_nvidia_image()
+                    unit_test_linux('pytorch', 'gpu_nv')
```

Comment:
We only need to test cugraph unittests. Perhaps also name the stage as `Torch+cuGraph`,